drivers/sensor/apds9253/apds9253.c

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ * Copyright (c) 2018 Phytec Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT avago_apds9253
+
+/* @file
+ * @brief driver for APDS9253 ALS/RGB/
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/pm/device.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <string.h>
+#include <zephyr/logging/log.h>
+
+#include "apds9253.h"
+
+#define BYTES_PER_VALUE   3
+#define VALUES_PER_SAMPLE 4
+
+LOG_MODULE_REGISTER(APDS9253, CONFIG_SENSOR_LOG_LEVEL);
+
+static inline void apds9253_setup_int(const struct apds9253_config *cfg, bool enable)
+{
+	unsigned int flags = enable ? GPIO_INT_EDGE_TO_ACTIVE : GPIO_INT_DISABLE;
```

Comment:
```suggestion
	gpio_flags_t flags = enable ? GPIO_INT_EDGE_TO_ACTIVE : GPIO_INT_DISABLE;
```

include/zephyr/drivers/gpio.h